### PR TITLE
Api1800 ethernet networks

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Api1800 support for ethernet networks

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [x] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -49,12 +49,13 @@
...
+|<sub>/rest/ethernet-networks/{id}</sub>                                                  | PATCH    | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |
+|<sub>/rest/ethernet-networks/{id}</sub>                                                  | DELETE   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/ethernet-networks/bulk-delete</sub>                                           | POST     | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   |
```
Comment (chebroluharika) [inline endpoints-support.md:58]: Make it green for 1600 also

**Review by chebroluharika — APPROVED** [review]
Please address 1 comment given before merge

File: endpoints-support.md (hunk 1 above)
Comment (VenkateshRavula) [reply endpoints-support.md:58]: we have added bulk-delete in api1800 files only.. so didn't add it for 1600

Comment (chebroluharika) [reply endpoints-support.md:58]: okay

**Review by yuvirani — APPROVED** [review]

**Review by AsisBagga — APPROVED** [review]
LGTM 👍

**Pull request merged**